src/Talis/Persona/Client/OAuthClients.php

```diff
@@ -93,7 +81,7 @@ public function updateOAuthClient($clientId, $properties, $token)
      */
     protected function personaPatchOAuthClient($url, $properties, $token)
     {
-        $curlOptions = array(
+        $this->performRequest(array(
```

Comment:
Just noticed that the `personaGetOAuthClient()` method returns this line, but this one doesn't.